pages/ksphere/kommander/latest/operations/cloud-providers/index.md

```diff
@@ -13,34 +13,15 @@ In order to provision new clusters and manage them, Kommander needs cloud provid
 
 Figure 6 - Cloud Provider Form
 
-### Configuring an AWS cloud provider
+## Configuring an AWS cloud provider
 
-When creating an AWS cloud provider, you need an access id and secret key for a user with a set of minimum  capabilities.
+When creating an AWS cloud provider, you can choose between `Static` authentication method and [Role-based authentication method][instance_profiles].
```

Comment:
unsure if we maybe should name it `Role based` instead of `Role-based`
but for sure `iam_roles` instead of `instance_profiles`🙏

Reply:
wiki says role-based https://en.wikipedia.org/wiki/Role-based_access_control :)